website/docs/developer/research/research-deep-glacier-migration.md

```diff
@@ -0,0 +1,154 @@
+---
+id: research-deep-glacier-migration
+title: Deep archive storage migration Research Notes
+description: Research Notes on deep archive migration.
+---
+
+## Overview
+
+Deep archive S3 storage type is the cheapest glacier storage type currently present in AWS S3. For archive data that does not require immediate access such as backup or disaster recovery use cases, choosing `Deep Archive Access` tier could be a wise choice.
+- Storage cost is ~$1/TB data per month. 
+- Objects in the Deep Archive Access tier are moved to the Frequent Access tier within 12 hours. Once the object is in the Frequent Access tier, users can send a GET request to retrieve the object.
+
+
+### Assumptions
+
+- List of objects already exists in orca catalog and S3 bucket.
+- Users should be able to provide some information(filename) to retrieve objects possibly at the collection level.
+
+### Initial Implementation Idea
+
+Migration should be a 2 step process- first retrieve the objects and then copy the objects to deep archive.
+
```

Comment:
A graphic would be good here showing the general use case and flow. You need to think a little bigger here like how will the end user initiate this migration and what steps will occur once they do? A [UML Sequence Drawing](https://developer.ibm.com/articles/the-sequence-diagram/) or [C4 Dynamic Diagram](https://c4model.com/#DynamicDiagram) would work well here.

Reply:
tried adding a drawing and relevant step fn workflow

Reply:
The fn workflow diagram does not add anything, I would create a UML sequence diagram to show how an end user would interact with the code or a C4 Dynamic diagram to better tell the story.